app/src/main/java/simform/gitexcercise/android/module/authentication/LoginActivity.kt

```diff
@@ -0,0 +1,61 @@
+package simform.gitexcercise.android.module.authentication
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.core.widget.addTextChangedListener
+import simform.gitexcercise.android.MainActivity
+import simform.gitexcercise.android.R
+import simform.gitexcercise.android.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityLoginBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        configureFields()
+        setOnClickListenersOnButtons()
+    }
+
+    private fun setOnClickListenersOnButtons() {
+        binding.btnLogin.setOnClickListener {
+            login()
+        }
+
+        binding.btnCreateAccount.setOnClickListener {
+            startActivity(Intent(this, RegistrationActivity::class.java))
+        }
+
+        binding.btnForgotPassword.setOnClickListener {
+            startActivity(Intent(this, ForgotPasswordActivity::class.java))
+        }
+    }
+
+    private fun configureFields() {
+        binding.etUsername.addTextChangedListener {
+            binding.tlUsername.isErrorEnabled = false
+            binding.tlUsername.error = null
+        }
+        binding.etPassword.addTextChangedListener {
+            binding.tlPassword.isErrorEnabled = false
+            binding.tlPassword.error = null
+        }
+    }
+
+    private fun login() {
```

Comment:
function name should be have some action intent for e.g doLogin(), registerUser() etc

Reply:
Resolved